setup.py

```diff
@@ -137,7 +136,10 @@ def download_url(version):
                    "License :: OSI Approved :: MIT License",
                    "Natural Language :: English",
                    "Operating System :: OS Independent",
-                   "Programming Language :: Python :: 2.7",
+                   "Programming Language :: Python :: 3",
+                   "Programming Language :: Python :: 3.5",
+                   "Programming Language :: Python :: 3.6",
+                   "Programming Language :: Python :: 3.7",
```

Comment:
Same as above: Why these versions? (I have no idea, but it should be explained.)

Reply:
changed to classifier to a non specific one so any py3 version works with pypi.